apps/mobile-wallet/src/components/BalanceSummary.tsx

```diff
@@ -67,7 +67,7 @@ export default BalanceSummary
 const BalanceSummaryContainer = styled.View`
   justify-content: center;
   align-items: center;
-  margin: 10px 0;
```

Comment:
I can't seem to be able to test the wallet connect modal on ios...


https://github.com/user-attachments/assets/22cb03da-3208-4c3e-9047-b3bd1c57d8d2